task/remote-scp/0.1/remote-scp.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: remote-scp
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.43.0"
```

Comment:
I guess `tekton.dev/v1` was introduced in `0.50.0` can we set that version here? 😅

Reply:
Modification completed, please check